tidy-data/01-Reshape-Data/01-Reshape-Data.Rmd

```diff
@@ -330,20 +334,19 @@ table4a %>% gather(key = "year", value = "cases", 2, 3)
 
 ### Specifying columns
 
-So far we've used numbers to describe which columns to reshape with `gather()`, but this isn't necessary. `gather()` also recognizes column names as well as all of the `select()` helpers that you learned about in [Isolating Data with dplyr](). So for example, these expressions would all do the same thing:
+So far we have specified explicitly which columns to pivot, but this isn't necessary. `pivot_longer()` also recognizes column names as well as all of the `select()` helpers that you learned about in [Isolating Data with dplyr](https://rstudio.cloud/learn/primers/2.2). So, for example, these expressions would all do the same thing: 
```

Comment:
Suggest simpler words, maybe like: "So far, we have listed which columns to pivot by naming them one at a time and combining them using the `c()` function, but this isn't necessary."

Reply:
👍